README.md

```diff
@@ -17,6 +17,8 @@
 
 This is a sample application demonstrating Quarkus features and best practices. The application allows superheroes to fight against supervillains. The application consists of several microservices, communicating either synchronously via REST or asynchronously using Kafka. All the data used by the applications are [on the `characterdata` branch](https://github.com/quarkusio/quarkus-super-heroes/tree/characterdata) of this repository.
 
+This is **NOT** the workshop with the step-by-step instructions. If you are looking for the Quarkus Super Heroes workshop, you can find it at https://github.com/quarkusio/quarkus-workshops/tree/main/quarkus-workshop-super-heroes.
```

Comment:
Would https://quarkus.io/quarkus-workshops/super-heroes/ be a better link for the workshop? @holly-cummins / @xstefank ?

Reply:
TBH, I have no preference. So I can change it if you think it's better.

Reply:
I'll let @holly-cummins weigh in.

Reply:
If people want the formatted instructions, Eric's link is better. And, thinking about it, people are most likely to want the formatted instructions. 

We should probably put a link from the formatted instructions back to the repo, for PRs + bug reports, but that's a separate (bigger) thing.

Reply:
When you get the formatted instructions there is already a link back to the repo:

![image](https://github.com/user-attachments/assets/7aabce73-8773-4ac5-86c9-42ae980e24e4)